build/Dockerfile

```diff
@@ -22,4 +22,8 @@ COPY job-templates/* /default-job-templates/
 # install operator binary
 COPY _output/bin/eunomia ${OPERATOR}
 
+RUN groupadd -r eunomia && useradd --no-log-init -r -g eunomia eunomia
+
 ENTRYPOINT ["/usr/local/bin/eunomia-operator"]
+
+USER eunomia
```

Comment:
I think `USER` needs to be before `ENTRYPOINT`. I think after that it should be good to merge.

From ... https://docs.docker.com/engine/reference/builder/#user ...

> The USER instruction sets the user name (or UID) and optionally the user group (or GID) to use when running the image and for any RUN, CMD and ENTRYPOINT instructions **that follow it** in the Dockerfile.

Reply:
Oooops, I think you might be right. Thank you for noticing.